uploader/serializers.py

```diff
@@ -1,9 +1,10 @@
 from rest_framework import serializers, fields
-
+from datetime
```

Comment:
aha. This is the issue. I dont think this would work. Probably you meant, `import datetime` ? And,

Reply:
Sorry for that, I have changed it.